### DISCUSSION TITLE
ReadWriteMany and PVC

### DISCUSSION THREAD

**Discussion by albundy83** [openappsec/openappsec#244] [Feature Requests]
Hello,

If nginx as DaemonSet need to be supported, you should update also [here](https://github.com/openappsec/openappsec/blob/main/build_system/charts/open-appsec-k8s-nginx-ingress/templates/appsec-pvc.yaml#L15) and [here](https://github.com/openappsec/openappsec/blob/main/build_system/charts/open-appsec-k8s-nginx-ingress/templates/appsec-pvc.yaml#L40) to be able to mount PVC on each DaemonSet.

Maybe you could also choose a way to name storageClass value, [here](https://github.com/openappsec/openappsec/blob/main/build_system/charts/open-appsec-k8s-nginx-ingress/templates/appsec-pvc.yaml#L20) it's `Values.appsec.persistence.storageClass`
and [here](https://github.com/openappsec/openappsec/blob/main/build_system/charts/open-appsec-k8s-nginx-ingress/templates/appsec-learning-pvc.yaml#L17) it's `.Values.appsec.persistence.learning.storageClass.name`.
Beside, in documentation [here](https://docs.openappsec.io/getting-started/start-with-kubernetes/install-using-helm-ingress-nginx-and-kong), you have an error as it's written:
```
If this value is set to true (default, when not overriding with false) you must also specify appsec.persistence.learning.storageClass
```
Maybe, it has caused the error from [here](https://github.com/openappsec/openappsec/issues/142).

It will be nice also to handle updateStrategy to Recreate when PVC is mounted instead of just RollingUpdate.

Thanks a lot

**Comment by orianelou** [comment]
Hi @albundy83, 

Thank you for the great suggestion! This is already in our road map and I'll update here once this will be supported,
